### PR TITLE
feat: create classroom

### PR DESCRIPTION
**Pull request by SiongSng**
# Description
Add create a classroom API

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/model/auth/connect_account.ts [hunk 1]
```diff
@@ -2,6 +2,10 @@
  * The connect account of the user for save the third party OAuth info (e.g. google, facebook, etc.).
  */
 export interface ConnectAccount {
+  /**
+   * The type of the connect account.
```
Comment (lumynou5) [inline src/model/auth/connect_account.ts]: 建議改成 type of connection

File: src/model/auth/connect_account.ts [hunk 2]
```diff
@@ -3,7 +3,7 @@
  */
 export interface ConnectAccount {
   /**
-   * The type of the connect account.
+   * Type of the connect account.
```
Comment (lumynou5) [inline src/model/auth/connect_account.ts]: 喔，我的意思是，connect 是動詞

**Review by lumynou5 — CHANGES_REQUESTED** [review]
`src/router/api/classroom/controller.ts` 有註解少句號。

**Comment by lumynou5** [comment]
`src/util/util.ts` 檔名是不是可以再好一點？

**Comment by SiongSng** [comment]
> `src/util/util.ts` 檔名是不是可以再好一點？

但這部份的內容都比較雜，我想不太到更好的名稱

**Comment by SiongSng** [comment]
> `src/router/api/classroom/controller.ts` 有註解少句號。

改好了

**Review by MKMonkeyCat — APPROVED** [review]

**Review by lumynou5 — APPROVED** [review]

**Comment by SiongSng** [comment]
ready for merge?

**Pull request merged**